src/api/app/mailers/event_mailer.rb

```diff
@@ -52,4 +52,20 @@ def event(subscribers, e)
       format.text { render template_name, locals: locals } if template_exists?("event_mailer/#{template_name}", formats: [:text])
     end
   end
+
+  private
+
+  # Reencode the fail log replacing invalid UTF-8 characters with the default unicode replacement character: '\ufffd'
+  # source: https://stackoverflow.com/a/24493972
+  def reencode_faillog(faillog)
+    faillog.encode!('UTF-8', 'UTF-8', invalid: :replace)
+  end
+
+  def sanitize_expanded_payload(payload)
+    if payload['faillog']
```

Comment:
@danidoni are the invalid characters only in the `faillog`? just by looking at the backtrace and issue it is not clear to me :)

Reply:
The invalid characters are only rendered in the email notifying of the failures, which only uses the `faillog`. So yes, only in the `faillog`.

Reply:
alright :+1: